src/utils/asHealingObject.js

```diff
@@ -0,0 +1,26 @@
+import { asMaybe, asObject } from 'cleaners'
+
+export function asHealingObject(shape, fallback) {
```

Comment:
I really don't like this pattern. Let's have a sidebar and find a less over engineered way to solve the type issue.
This is like building a tank to kill some ants.....